### PR TITLE
[UXENG-2369] Add Segment to track dev-center via Google Analytics

### PR DESCRIPTION
**Pull request by eliperelman**
Fixes https://algorithmia.atlassian.net/browse/UXENG-2369

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -78,7 +78,9 @@
     <!-- BEGIN_SCRIPTS_EMBED -->
     {% unless site.enterprise %}
       <script src="{{ site.baseurl }}/js/sentry.5.7.1.min.js" type="text/javascript" crossorigin="anonymous" charset="utf-8"></script>
+      <script src="{{ site.baseurl }}/js/segment.min.js" type="text/javascript" crossorigin="anonymous" charset="utf-8"></script>
```
Comment (ryanmillerdev) [inline _layouts/default.html]: ⚠️ Firing this up locally, I see `/js/segment.js` resolve successfully, but this file `404`s.

File: js/segment.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+!function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
+  analytics.load('UA-38617973-4');
```
Comment (ryanmillerdev) [inline js/segment.js]: ⚠️ You'll actually want this to the write key for your new Segment source, versus the UA ID from Google analytics. I'd be happy to walk you through creating a new Segment source, and here are more docs on `analytics.js`: https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/quickstart/.

**Review by ryanmillerdev — DISMISSED** [review]
Looks like the call to fetch `analytics.js` is 404ing due to the usage of the Google Analytics UA ID, versus a Segment write ID. Headed on a walk with my kids shortly, but happy to hop on a call this afternoon and show you the Segment dashboard in greater detail.

File: _layouts/default.html (hunk 1 above)
Comment (eliperelman) [reply _layouts/default.html]: Oops, forgot to include this in my commit; I'll get this pushed up.

**Review by bnlb — APPROVED** [review]
My local environment is all messed up at the moment due to Ruby issues I've been putting off fixing, but code changes LGTM.

I'd wait for @andryanmiller to confirm before merging.

**Review by ryanmillerdev — APPROVED** [review]
Looks good! Events are firing as expected.

**Pull request merged**